push-files/push.test.js

```diff
@@ -42,11 +34,17 @@ test('builds correct metadata', async () => {
 });
 
 test('generates a content hash', async () => {
-  const hash = await getHashOfFiles('./dir');
+  const hash = await getHashOfFiles('./fixtures');
```

Comment:
I started out with mocking `walk` `crypto` but as soon as I was about to mock `fs` I decided to use a fixtures folder instead of mocking

Reply:
💯